Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Plan.hs

```diff
@@ -374,33 +374,34 @@ createMandateInvoiceAndOrder driverId merchantId plan = do
     Just registerFee -> do
```

Comment:
Just check here if registerFee.platformFee + currentDues are `0` then INACTIVE the registerFee and create a new one with `1` rupee platform fee.